app/services/metrics/base.rb

```diff
@@ -0,0 +1,28 @@
+module Metrics
+  class Base < BaseService
+    def calculate_avg(entities, metric_type)
```

Comment:
Perhaps we can rename this to `calculate_metrics_avg` or something like that, so it's not confused with the other method, `calculate_average`